engines/ags/shared/core/platform.h

```diff
@@ -129,14 +129,6 @@ namespace AGS3 {
 #error "No endianness defined"
 #endif
 
-#if defined(_DEBUG)
-#define AGS_PLATFORM_DEBUG  (1)
-#elif ! defined(NDEBUG)
-#define AGS_PLATFORM_DEBUG  (1)
```

Comment:
This one should be kept, if I understand correctly.
AGS_PLATFORM_DEBUG is still checked in `shared/util/file.cpp`